internal/webservice/webservice.go

```diff
@@ -19,10 +21,13 @@ type WebService struct {
 	eventRepository repository2.EventRepository
 	healthService   health.Interface
 	assetsDir       string
+	corsConfig      cors.Options
+	router          *httprouter.Router
```

Comment:
Might be better not to keep the router in a struct since the only use of it is to build routes and install it on `ws.server.Handler`